shell.nix

```diff
@@ -4,55 +4,68 @@ let
   fhs = pkgs.buildFHSUserEnv {
     name = "fiedka";
     targetPkgs = pkgs: with pkgs; [
+      nodejs
+      go_1_17
+      git
+
+      # common requirements
+      freetype
+      fontconfig
+      glib
+      gtk2
+      gtk3
+
       # Copied from another Electron application. Probably doesn't need everything
       bash
       coreutils
       zlib
       stdenv.cc.cc
-      xorg.libXext
-      xorg.libX11
+      xorg.libXft
+
+      # Copied from atom
+      dbus
+      atk
+      at-spi2-atk
+      at-spi2-core
+      cairo
+      pango
+      libgnome-keyring3
+      gdk-pixbuf
+      cups
+      expat
+      nspr nss
+
       xorg.libXrender
+      xorg.libX11
+      xorg.libXext
+      xorg.libXdamage
       xorg.libXtst
+      xorg.libXcomposite
       xorg.libXi
-      xorg.libXft
+      xorg.libXfixes
+      xorg.libXrandr
+      xorg.libXcursor
+      xorg.libxkbfile
+      xorg.libXScrnSaver
       xorg.libxcb
-      # common requirements
-      freetype
-      fontconfig
-      glib
-      gtk2
-      gtk3
 
-      nodejs
-      go_1_17
-      git
-      #electron
+      libcap
+      systemd
+      libnotify
+      libsecret
+      libuuid
+      libdbusmenu
+      libdrm
+      mesa # for libgbm
 
       # Manually added
       xorg.libxshmfence
-      nss
-      nspr
-      atk
-      at-spi2-atk
-      dbus
-      libdrm
```

Comment:
I put them here under this comment so we know these are strictly required.
The ones at the top, copied from atom are probably not all required. So those we have to double-check, these here we don't.

Reply:
Makes sense, I'll move those again, thank you!